gocourse18/service/service.go

```diff
@@ -0,0 +1,42 @@
+package service
+
+import (
+	"encoding/json"
+	"github.com/vikibell/prjctr-golang-beginning/gocourse18/domains/user"
+	"io"
+	"log/slog"
+	"os"
+	"path/filepath"
+)
+
+func GetUsersFromFile() []user.User {
```

Comment:
Краще повертати помилку і логувати десь на рівень вище:
```suggestion
func GetUsersFromFile(filepath string) ([]user.User, error) {
```